Find a functionally equivalent code:android.appwidget.AppWidgetHostView.updateAppWidget:COMMENT
Method Modifier: public      
Comment:/**
 * Process a set of {@link RemoteViews} coming in as an update from the
 * AppWidget provider. Will animate into these new views as needed
 */

@@ -33,39 +33,39 @@
     } else {
         // Prepare a local reference to the remote Context so we're ready to
         // inflate any requested LayoutParams.
-        mRemoteContext = getRemoteContext(remoteViews);
+        mRemoteContext = getRemoteContext();
         int layoutId = remoteViews.getLayoutId();
         // layout matches, try recycling it
         if (content == null && layoutId == mLayoutId) {

